src/app/shared/dialogs/dialogs-form.service.ts

```diff
@@ -11,11 +11,11 @@ import {
 export class DialogsFormService {
   constructor(private dialog: MatDialog, private fb: FormBuilder) { }
 
-  public confirm(title: string, fields: any, formGroup: any): Observable<boolean> {
+  public confirm(title: string, fields: any, formGroup: any, autoFocus?: false): Observable<boolean> {
```

Comment:
@singharpita This syntax is incorrect.  To set a default value for a function parameter you should use an `=` and no `?` as in:

```
autoFocus = false
```